modin/config/envvars.py

```diff
@@ -404,6 +404,18 @@ def get(self):
         return result
 
 
+class MinNumElementsToStartNewPandasPartition(EnvironmentVariable, type=int):
+    """
+    Minimum number of rows/columns in a single pandas partition split.
+
+    Once a partition for a pandas dataframe has more than this many elements,
+    Modin adds another partition.
+    """
+
+    varname = "MODIN_MIN_NUM_ELEMENTS_TO_START_NEW_PANDAS_PARTITION"
```

Comment:
```suggestion
    varname = "MODIN_MIN_ELEMENTS_IN_PARTITION"
```
I don't see how users would set such a long name... let's make it shorter.

This should probably be reflected in the class name, too.

Reply:
Done.